galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.jmeter.manager.ivt/src/main/resources/resources/jmeter.properties

```diff
@@ -0,0 +1,1346 @@
+################################################################################
+# Apache JMeter Property file
+################################################################################
+
+##   Licensed to the Apache Software Foundation (ASF) under one or more
+##   contributor license agreements.  See the NOTICE file distributed with
+##   this work for additional information regarding copyright ownership.
+##   The ASF licenses this file to You under the Apache License, Version 2.0
+##   (the "License"); you may not use this file except in compliance with
+##   the License.  You may obtain a copy of the License at
+##
+##       http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This could be a problem as the license of the property file is APL2.0 and we are re-shipping, I will check with legal

Reply:
This specific properties file came from a example properties file by jmeter but can be substituted with self made property files if that would solve this issue

Reply:
I've checked.  It will be pulled up in the CSAR scan but I can just mention it we can move on.  It's not a problem